dev-python/pydotplus/files/pydotplus-2.0.2-tests.patch

```diff
@@ -0,0 +1,26 @@
+--- a/test/pydot_unittest.py
++++ b/test/pydot_unittest.py
+@@ -18,7 +18,7 @@ import sys
+
+ import pydotplus
+ import unittest
+-
++import pytest
```

Comment:
Sorry I didn't notice this before. So you're actually adding dependency on pytest? Why aren't you just using unittest's functions?